.travis.yml

```diff
@@ -1,14 +1,13 @@
 language: scala
 scala:
-- 2.13.0-M3
-- 2.12.4
+- 2.13.0-M4
+- 2.12.6
 - 2.11.12
-- 2.10.7
 jdk:
 - oraclejdk8
 - oraclejdk9
 script:
-- sbt -DscalaJSStage=full ++$TRAVIS_SCALA_VERSION validateCode mimaReportBinaryIssues testOnly
```

Comment:
We should either start to release play-json 2.7.0 and filter the binary compatibility issues, or we will need to have `scala-2.13.0-M4` to handle the necessary changes.

WDYT?

Reply:
if I understand the problem correctly, you could do `mimaPreviousVersion := (CrossVersion.partialVersion(scalaVersion.value) match { case Some((2, n)) if n <= 12 => Some(...); case _ => None } // TODO update once Scala 2.13.0 final is released`, that kind of thing? if `mimaPreviousVersion` is `None` then `mimaReportBinaryIssues` prints a warning but doesn't fail the build.

Reply:
Noticed this is already handled here:

https://github.com/playframework/play-json/blob/80f42eab24f0c14f6569456d8dd602b9367e6e98/build.sbt#L47-L54

So nothing to do besides add mima filters and add the task back in Travis config.